### PR TITLE
[DO NOT MERGE]CTV-128 - Add LOA field

### PR DESCRIPTION
**Pull request by Rosa-Fox**
- Add radio buttons for user to select `LOA1` or `LOA2`.

Please note; the form hint text is a placeholder and should be approved by a tech writer.

### CODE REVIEW

File: app/views/environment_config_form/environment_config_form.html.erb [hunk 1]
```diff
@@ -454,6 +454,25 @@
 
     </fieldset>
 
+    <fieldset class="level-of-assurance">
```
Comment (karlbaker02) [inline app/views/environment_config_form/environment_config_form.html.erb]: This should be the first field on the form (before Verify service entity ID)

File: app/views/environment_config_form/environment_config_form.html.erb [hunk 2]
```diff
@@ -454,6 +454,25 @@
...
+      <h2 class="heading-medium">Level of assurance</h2>
+        <span class="form-hint">
+          A 'level of assurance' determines how much information is needed for verification.
+          <br>
+          To find out more please visit <a href="https://govuk-verify.cloudapps.digital/get-started-understand-how-verify-works/">this guide</a>.
+          <br>
+          <a href="mailto:idasupport+onboarding@digital.cabinet-office.gov.uk">Email us</a> if you need help deciding which LOA is right for you.
+        </span>
+      <div class="multiple-choice">
+        <%= f.radio_button :level_of_assurance, 'LOA-1' %>
```
Comment (karlbaker02) [inline app/views/environment_config_form/environment_config_form.html.erb]: We should maybe select one of these options by default too?

File: app/models/onboarding_form_service.rb [hunk 3]
```diff
@@ -113,6 +113,9 @@ def generate_ticket_body(onboarding_form)
                 Department:
                 #{ value_or_default(onboarding_form.contact_details_department) }
 
+                Level of assurance:
```
Comment (karlbaker02) [inline app/models/onboarding_form_service.rb]: We should move this up to the same place as we display it in the form too, so it's easier to know where to look for it.

**Comment by Rosa-Fox** [comment]
@karlbaker02 thanks for the feedback, all good suggestions. I have made the changes and amended the commit.

**Comment by karlbaker02** [comment]
Looks good from a tech point-of-view. Will add Tom so he can take a quick look at the content / styling of the LOA, then will approve.

**Pull request merged**